llvm/tools/repo2obj/R2OELFOutputSection.h

```diff
@@ -240,8 +240,10 @@ OutputSection<ELFT>::SectionInfo::symbol(SymbolTable<ELFT> &Symbols,
     static auto PrivateSymbolCount = 0U;
 
     auto Name = Generated.add(".LR" + std::to_string(PrivateSymbolCount++));
-    Symbol_ = Symbols.insertSymbol(Name, Section_, Offset_, 0 /*size*/,
-                                   pstore::repo::linkage_type::internal);
+    Symbol_ =
+        Symbols.insertSymbol(Name, Section_, Offset_, 0 /*size*/,
+                             pstore::repo::linkage_type::internal,
+                             pstore::repo::visibility_type::default_visibility);
```

Comment:
I think that generated symbols can have “hidden” visibility.

Reply:
In the LLParser.cpp , `isValidVisibilityForLinkage` function check a valid visibility during parsing the Gobal. It mentioned `symbol with local linkage must have default visibility`.